src/main/java/com/splunk/kafka/connect/SplunkSinkTask.java

```diff
@@ -230,6 +232,7 @@ public String headerId(SinkRecord sinkRecord) {
         if(sourceHeader != null) {
             headerString.append(sourceHeader.value().toString());
         } else {
+            if(metas!=null)
```

Comment:
The if can be formatted better with braces

Reply:
Added braces